pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[tool.poetry]
+name = "healthrex_ml"
+version = "0.1.0"
+description = "Tools for creating cohorts, features, and models"
+authors = ["Conor Corbin <ccorbin@stanford.edu>"]
+
+[tool.poetry.dependencies]
+python = "^3.7.1,<3.11"
+google-cloud-bigquery = "3.3.2"
+pandas = "1.3.5"
+scikit-learn = "1.0.2"
+torch = "1.12.1"
+tensorboard = "2.10.1"
+tqdm = "4.64.1"
+pandas-gbq = "0.17.9"
+grpcio-tools = "1.44.0" # for weird protobuf errors
+seaborn = "0.12.1"
+pyyaml = "6.0"
+lightgbm = "3.1.1"
+
+[tool.poetry.group.dev.dependencies]
+jupyter = "1.0.0" # likely useful
```

Comment:
Let's move Jupyter up to tool.poetry.dependencies so that installs with pip.

Reply:
I'll make the change today and re-review -- getting hung up setting up a new Macbook and ran into issues with Poetry when using Python 3.11 (since the lightgbm dependency only goes up to Python 3.10). Want to think through some other files to add / commit to make it easier though should be able to resolve this today!